FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.module.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  width: 22.125rem;
+  height: 11.125rem;
+  border-radius: 0.5rem;
+  overflow: hidden;
+  border: 0.25rem solid #f8e6d8 !important;
+}
+
+.firstCol {
+  width: 17.625rem;
+  height: 11rem;
+
+  display: flex;
+  order: 0;
+  flex-direction: column;
+
+  border-right: 0.5rem solid #f8e6d8;
+  box-sizing: border-box;
+}
+
+.information {
+  padding: 1.5rem 0.5rem 1.5rem 2.25rem;
+}
+
+.nameinfo {
+  flex: display;
+  margin: 0rem 0.5rem;
+  align-self: center;
+  letter-spacing: 0.03em;
+
+  .name {
+    color: #000000;
+  }
+
+  .title {
+    color: rgba(32, 32, 32, 0.6);
+  }
+}
+
+.caption {
+  color: rgba(32, 32, 32, 0.87);
+  letter-spacing: 0.025em;
+  margin: 1rem 0rem;
+}
+
+.secondCol {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+
+  width: 4.5rem;
+  height: 11.125rem;
+
+  box-sizing: border-box;
+}
+
+.design {
+  background: #a5bf84;
+}
+
+.engineer {
+  background: #e7946d;
+}
+
+.business {
+  background: #6d83e7;
+}
+
+.picture {
+  border: $default-border-width solid $primary-light-color;
+  border: 0.0229rem solid #e7946d;
```

Comment:
Why do you have 2 `border: ` css lines right after one another?
Also, try not to use magic numbers in border width and radius also (if possible). If those might be reused elsewhere, consider making them global vars or check if they're already defined. If they're only to be used in this file (not sure about this), then you can keep them as magic numbers